packages/story-editor/src/components/library/panes/media/local/hotlink/useInsert.js

```diff
@@ -79,8 +79,8 @@ function useInsert({ link, setLink, setErrorMsg, onClose }) {
       } = hotlinkData;
 
       try {
-        const proxiedUrl = getProxiedUrl({ isExternal: true }, link);
-        const resource = await getResourceFromUrl(proxiedUrl, type);
+        const proxiedUrl = getProxiedUrl({ needsProxy }, link);
```

Comment:
Nit: Is there a reason to call `getProxiedUrl` at all here if it `needsProxy` is `false`?

Reply:
Good point. This can be simplified.